src/modules/menu/MenuModule.vue

```diff
@@ -1,46 +1,54 @@
 <template>
     <transition name="slide-up">
-        <div v-show="showHeader" class="header align-items-center p-1 flex-fill">
-            <SwissFlag class="swiss-flag ml-1 mr-2" />
-            <HeaderSwissConfederationText class="d-none d-sm-block" />
-            <!-- we then let whatever was given in the slot be rendered here, that's where we expect to receive the search module from MapView.vue -->
-            <slot />
-            <HeaderMenuButton />
-            <MenuTray class="menu-tray" />
+        <div v-show="showHeader" class="header" :class="{ 'extra-height': showLoadingBar }">
+            <div class="header-content align-items-center p-1 flex-fill">
+                <SwissFlag class="swiss-flag ml-1 mr-2" />
+                <HeaderSwissConfederationText class="d-none d-sm-block" />
+                <!-- we then let whatever was given in the slot be rendered here, that's where we expect to receive the search module from MapView.vue -->
+                <slot />
+                <HeaderMenuButton />
+                <MenuTray class="menu-tray" />
+            </div>
+            <HeaderLoadingBar v-if="showLoadingBar" />
         </div>
     </transition>
 </template>
 
 <style lang="scss">
 @import 'node_modules/bootstrap/scss/bootstrap';
+@import 'src/scss/variables';
 @import 'src/scss/media-query.mixin';
 
-$headerHeight: 3rem;
-
 .header {
     position: fixed;
     top: 0;
     left: 0;
-    height: $headerHeight;
+    height: $header-height;
     width: 100%;
     background: $white;
-    border-bottom: 4px solid $red;
-    display: flex;
+    box-shadow: 6px 6px 12px rgb(0 0 0 / 18%);
     // so that the menu is above the map overlay
     z-index: 500;
-    .swiss-flag {
-        height: 2rem;
-        width: 2rem;
-        min-height: 2rem;
-        min-width: 2rem;
+    &.extra-height {
+        height: $header-height + $header-loading-bar-height;
     }
-    .menu-tray {
-        position: fixed;
-        top: $headerHeight;
-        right: 0;
-        background: $white;
-        width: 95%;
-        max-width: 40rem;
+    .header-content {
+        display: flex;
+        height: $header-height;
```

Comment:
Just for my understanding, why changing the naming convention here ? is `$header-height` a javascript variable, Vue variable ? For variable I would prefer camelCase or snake_case as kebab case

Reply:
that's a SCSS variable (Sass), and as everything in CSS is with this style of notation I've changed it to that. You can find all our variables in `src/scss/variables.scss`